File: backend/app/domain/entities/PointSaleEmail.py
from sqlalchemy import Column, Integer, String, Boolean, DateTime
from app.infrastructure.db.connection import Base

class PointSaleEmail(Base):
    __tablename__ = "PointSaleEmail"

    IdPointSaleEmail = Column(Integer, primary_key=True, index=True, autoincrement=True)
    emailPointSale = Column(String(150), nullable=False, unique=True)
    statusPointSaleEmail = Column(Boolean, nullable=False, default=True)
    createdAtPointSaleEmail = Column(DateTime, nullable=False)
    updatedAtPointSaleEmail = Column(DateTime, nullable=True)

File: frontend/src/models/PointSaleEmail.ts
export interface PointSaleEmail {
  IdPointSaleEmail: number;
  emailPointSale: string;
  statusPointSaleEmail: boolean;
  createdAtPointSaleEmail: string;
  updatedAtPointSaleEmail: string | null;
}

export interface PointSaleEmailCreate {
  emailPointSale: string;
}

export interface PointSaleEmailUpdate {
  emailPointSale?: string;
  statusPointSaleEmail?: boolean;
}